packages/pigment-css-react/README.md

```diff
@@ -854,3 +854,216 @@ function App() {
   )
 }
 ```
+
+## Building a UI library
```

Comment:
```suggestion
## Building a reusable component
```

Seems more fitting here? The idea is to teach how to build a component that can have a styling-related prop interface using Pigment, so I'd probably go somewhere this route!

Reply:
Using "UI library" is more appropriate for this section. "reusable component" is too broad, developers can create "reusable component" with a simple `styled('div')(…)`.

This section is specific to design system authors who want to build a library that will be shared among projects. cc @samuelsycamore to help on this.

Reply:
> This section is specific to design system authors who want to build a library that will be shared among projects.

```suggestion
## Building a design system component
```

Something along these lines? I find the "Building a UI library" title weird because we're demonstrating how to build just one component, not all that you need for a library. But I see what you mean; it's like a "design system-ready component", more than a simple reusable one.